opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -482,42 +486,50 @@ message DoubleHistogramDataPoint {
+  // buckets is one of several defined bucketing strategies
   //
-  // The number of elements in bucket_counts array must be by one greater than
-  // the number of elements in explicit_bounds array.
-  repeated fixed64 bucket_counts = 6;
+  // If this field is omitted, there is an implied single bucket
+  // spanning (-Inf,+Inf) containing the whole population.
+  oneof buckets {
+    // ExplicitBuckets support encoding arbitrary boundaries and
+    // counts.
+    ExplicitBuckets explicit = 9;
+  }
 
-  // A histogram may optionally contain the distribution of the values in the population.
-  // In that case one of the option fields below and "buckets" field both must be defined.
-  // Otherwise all option fields and "buckets" field must be omitted in which case the
-  // distribution of values in the histogram is unknown and only the total count and sum are known.
+  // (Optional) List of exemplars collected from
+  // measurements that were used to form the data point
+  repeated DoubleExemplar exemplars = 8;
+}
 
-  // explicit_bounds is the only supported bucket option currently.
-  // TODO: Add more bucket options.
+// ExplicitBuckets is a standard way of representing arbitrary
+// histogram data using explicit boundaries.  It combines an
+// array of N-1 boundaries and an array of N counts.
+message ExplicitBuckets {
+  // counts is an optional field contains the count values of histogram
+  // for each bucket.
+  //
+  // The sum of the counts must equal the value in the HistogramDataPoint's
+ count field.
+  //
+  // The number of elements in counts array must be one greater than
+  // the number of elements in boundaries array.
+  repeated fixed64 counts = 6;
 
-  // explicit_bounds specifies buckets with explicitly defined bounds for values.
+  // boundaries specifies buckets with explicitly defined boundaries.
   //
-  // This defines size(explicit_bounds) + 1 (= N) buckets. The boundaries for
+  // This defines N = size(boundaries) + 1 buckets. The boundaries for
   // bucket at index i are:
   //
-  // (-infinity, explicit_bounds[i]] for i == 0
-  // (explicit_bounds[i-1], explicit_bounds[i]] for 0 < i < N-1
-  // (explicit_bounds[i], +infinity) for i == N-1
+  // (-infinity, boundaries[i]] for i == 0
+  // (boundaries[i-1], boundaries[i]] for 0 < i < N-1
+  // (boundaries[i], +infinity) for i == N-1
   // 
-  // The values in the explicit_bounds array must be strictly increasing.
+  // The values in the boundaries array must be strictly increasing.
   //
   // Histogram buckets are inclusive of their upper boundary, except the last
   // bucket where the boundary is at infinity. This format is intentionally
   // compatible with the OpenMetrics histogram definition.
-  repeated double explicit_bounds = 7;
-
-  // (Optional) List of exemplars collected from
-  // measurements that were used to form the data point
-  repeated DoubleExemplar exemplars = 8;
+  repeated double boundaries = 7;
```

Comment:
For reference, that's https://github.com/OpenObservability/OpenMetrics/blob/142172882e57a52a26ec32ddfb43b15c42d7f520/proto/openmetrics_data_model.proto#L141

I have a slight preference for two parallel arrays, because the boundaries are constant and they ought to compress better (or could be pre-serialized). I'm definitely open to what the community wants, here.

Reply:
Given examplar's already are different from OM (with trace_id embedded too), then I think we'll have special handling here either way. 

I agree w/ @jmacd that repeated primitives may (or can) compress better.